z3c/dependencychecker/dotted_name.py

```diff
@@ -1,8 +1,18 @@
 # -*- coding: utf-8 -*-
 import hashlib
+import re
 from functools import total_ordering
 
 
+# some pypi packages' names have a prefix that is not part of the package
+# namespace
+# They can be safely removed
```

Comment:
My point was that they *cannot* be safely removed. Here are some paths from the first random django project I looked at:

 '/code/eggs/django_tls-0.0.2-py3.5.egg',
  '/code/eggs/django_oidc_provider-0.5.2-py3.5.egg',
  '/code/eggs/django_nose-1.4.4-py3.5.egg',
  '/code/eggs/django_extensions-1.6.7-py3.5.egg',
  '/code/eggs/django_appconf-1.0.2-py3.5.egg',

Of those five, three should get the django prefix removed, two (django-extensions and django-nose) should keep their prefix.